docs/knowledge_base/policy_development.md

```diff
@@ -108,3 +138,130 @@ If policy evaluation is successful, the result will contain both the policy outp
 Issues and dependencies that have been suppressed via project preferences are visible in the policy input, but rejections related to those issues or dependencies will not be included in the Markdown report.
 
 Dependencies that are ignored via the `ignored_packages` parameter are filtered out before applying the policy and will not be visible in the policy input or output.
+
+# Policy Examples
+
+The policy transforms your threat model into a description of why the job is being blocked. There are multiple ways to define why a job should be blocked.
+
+The `METADATA` block contains `OPA` [Annotations](https://www.openpolicyagent.org/docs/latest/annotations/) which correlate to the schema and can be used for type checking.
```

Comment:
Here way say that the annotations can be used for type checking, but we don't give an example of that. It still works, but I removed it from the examples when I migrated them to policy.v1 because it seemed like more clutter than it was worth. I probably missed updating this sentence.

Reply:
Feel free to make any updates you see fit, you know the policy development better than anyone else.

Reply:
I am not sure what to change here and you mentioned it still works, so I am going to resolve this and merge the PR.

My ask is that you open a new PR to fix up anything you think could be said in a more useful or correct way to the user. Particularly on the `policy_development.md` page.